### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Monitoring_Pause,Stop or Hide:There is no method provided for the users to pause,stop or hide the moving section of the graph

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#92]
**Environment Details:**
URL: https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS2.2.2

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Monitoring and press enter
6) Validate that any moving, blinking or scrolling information that starts automatically, lasts for more than five seconds, and is presented with other content provides a method for users to pause, stop, or hide it.

**Actual Result:**
There is no method provided for the users to pause, stop or hide the moving section of the graph

**Expected Result:**
Any moving, blinking or scrolling information that starts automatically, lasts for more than five seconds, and is presented with other content should provide a method for users to pause, stop, or hide it.

**User Impact:** 
There should be a mechanism for the user to pause, stop or hide the content unless the movement, blinking, or scrolling is part of an activity where it is essential
User specially the visually impaired user might face difficulty while going through the moving contents if there will not be any method of pausing or stopping the content

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b817da77-e26c-4651-9b98-eb38b304c8e0}

**Attachments:**
[Bug_92_No method provided for pause, stop and hide.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323481/Bug_92_No.method.provided.for.pause.stop.and.hide.pptx)

**Comment by jhkimnew** [comment]
The only reason of the monitoring feature page is to show those graphs. So, it does not make sense that users enter to the monitoring feature page and hide those graphs. In addition, users don't go to the monitoring feature page and get the same information with other methods.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.2.2; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
